package.json

```diff
@@ -40,11 +40,11 @@
     "sentry": "scripts/sentry.sh",
     "start": "scripts/start.sh",
     "start:dev": "concurrently 'yarn start:dev:relay' 'yarn start:dev:server'",
-    "start:dev:server": "NODE_ENV=development node --max_old_space_size=3072 -r dotenv/config ./src/dev.js",
```

Comment:
I've removed a number of `dotenv` preloads and instead import/require `loadenv` within scripts. I tested and they seemed to work. Please let me know if this may cause issues i've not accounted for.